src/cookbook/editable-svg-icons.md

```diff
@@ -67,32 +67,32 @@ export default {
 }
 ```
 
-The `currentColor` property that's the default on the fill will make the icon inherit the color of whatever text surrounds it. We could also pass in a different color as a prop if we wish.
+La propiedad `currentColor`, que es el valor por defecto en el _fill_, hará que el ícono herede el color de el texto que lo rodea. También podríamos pasar un color diferente como _prop_ si así lo deseamos.
 
-We can use it like so, with the only contents of `IconWrite.vue` containing the paths inside the icon:
+Lo podemos usar de la siguiente manera, teniendo como único contenido de `IconWrite.vue` los _paths_ dentro del ícono:
 
 ```html
 <icon-base icon-name="write"><icon-write /></icon-base>
 ```
 
-Now, if we'd like to make many sizes for the icon, we can do so very easily:
+Ahora, si quisieramos tener varios tamaños para el ícono, podríamos hacerlo facilmente:
 
 ```html
 <p>
-  <!-- you can pass in a smaller `width` and `height` as props -->
+  <!-- puede pasar valores más pequeños para `width` y `height` como props -->
   <icon-base width="12" height="12" icon-name="write"><icon-write /></icon-base>
-  <!-- or you can use the default, which is 18 -->
+  <!-- o puede usar el valor default, el cual es 18 -->
```

Comment:
Same, here, I think we can use `por defecto`
```suggestion
  <!-- o puede usar el valor por defecto, el cual es 18 -->
```

Reply:
Sounds good! Thanks @Tintef